examples/field_formats_example/common/example_currency_format.ts

```diff
@@ -34,6 +34,7 @@ export class ExampleCurrencyFormat extends FieldFormat {
     return new Intl.NumberFormat(undefined, {
       style: 'currency',
       currency: this.param('currency'),
+      currencyDisplay: 'narrowSymbol',
```

Comment:
Hi @eokoneyo! What triggered this change?

Reply:
Hey @jughosta! This change was added to fix a failure in this test [file](https://github.com/elastic/kibana/blob/f7bd91763f7b87e9f3528cee0bff2e94b192eef8/test/examples/field_formats/index.ts)

Context: I suppose because we pass the value `undefined` as the locale, this most likely cause the locale set to be the runtime's locale (this is an assumption based on [this](https://tc39.es/proposal-intl-numberformat-v3/out/numberformat/diff.html#sec-initializenumberformat)), which in turn causes inconsistencies in the result generated when the given value is formatted especially that the default currencyDisplay is the value `symbol`, more details [here](https://github.com/tc39/ecma402/issues/200). As far as I can tell passing the value `narrowSymbol` would not trigger the RangeError mentioned in the referenced issue since it's already widely supported.